platform/rct/hal/imu_stm32/imu_device_stm32.c

```diff
@@ -39,7 +39,7 @@
 #define IMU_TASK_PRIORITY	(tskIDLE_PRIORITY + 2)
 #define IS_9150_ADDR            0x68
 
-static struct is9150_all_sensor_data sensor_data[2];
+static struct is9150_all_sensor_data sensor_data[2] = {0};
```

Comment:
Ok -  if GCC will automatically juggle memory to fit in all available dis-contiguous BSS segments then I will give that a shot.  thanks!